TKGame/Player.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.VisualBasic;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Media;
+
+namespace TKGame
+{
+    class Player : Entity
+    {
+        private static Player instance;
+        public static Player Instance
+        {
+            get
+            {
+                // Creates the player if it doesn't already exist
+                if (instance == null)
+                    instance = new Player();
+
+                return instance;
+            }
+        }
+
+        private Player()
```

Comment:
Try to add some summary comments so the rest of us understand the point of your functions  and variables when using them